src/index.js

```diff
@@ -1,15 +1,11 @@
 import { PageMod } from 'sdk/page-mod';
 import self from 'sdk/self';
-import { storage } from 'sdk/simple-storage';
 import webext from 'sdk/webextension';
-import tabs from 'sdk/tabs';
 import { getMostRecentBrowserWindow } from 'sdk/window/utils';
 
-import { e10sStatus, e10sProcessCount } from './lib/e10s-status';
 import Logger from './lib/log';
+import handleMessage from './lib/messages';
```

Comment:
Why export handleMessage as default and another function from this module as a named export? Seems less confusing just to export handleMessage as a named export.